test/pgtest/ddl-extended.pt

```diff
@@ -1,3 +1,25 @@
+# Test that multiple DDL statements in the same extended request are supported
+send
+Parse {"query": "CREATE TABLE a (a int)"}
+Bind
+Execute
+Parse {"query": "CREATE TABLE b (a int)"}
+Bind
+Execute
+Sync
+----
+
+until
+ReadyForQuery
+----
+ParseComplete
+BindComplete
+CommandComplete {"tag":"CREATE TABLE"}
+ParseComplete
+BindComplete
+CommandComplete {"tag":"CREATE TABLE"}
+ReadyForQuery {"status":"I"}
```

Comment:
I'm pretty sure we're actually broken here. Try:

```
send
Parse {"query": "CREATE TABLE a (a int)"}
Bind
Execute
query": "ROLLBACK"}
Bind
Execute
Parse {"query": "CREATE TABLE b (a int)"}
Bind
Execute
Sync
----

until
ReadyForQuery
----
```

I think our output will differ from postgres' for this because we don't support DDL in transactions, but instead silently auto commit them.

Reply:
Oh wait no we're ok because you can eagerly commit in extended! So maybe we just differ from postgres, but in a spec-compliant way.